executors/rust/1.3/src/main.rs

```diff
@@ -17,23 +17,22 @@
 // https://unicode-org.github.io/icu4x-docs/doc/icu_collator/index.html
 
 mod collator;
+mod decimalfmt;
+mod displaynames;
```

Comment:
@sven-oly Are these files being used? I think they might be dead code because we have other modules, `numberfmt` and `langnames`, that do these tests.

Reply:
If they are dead code, please remove. Thanks, Rob, for this work!